Frameworks/CoreMotion/CMMotionManager.mm

```diff
@@ -190,18 +180,18 @@ - (void)startAccelerometerUpdatesToQueue:(NSOperationQueue*)queue withHandler:(C
         self.accelerometerQueue = queue;
 
         self.accelerometerToken =
-            [self.accelerometer addReadingChangedEvent:^void(WDSAccelerometer* sender, WDSAccelerometerReadingChangedEventArgs* e) {
```

Comment:
When ARC is enabled (as it is here), objects captured by lambdas should be retained.

---
In reply to: [111273056](https://github.com/Microsoft/WinObjC/pull/2505#discussion_r111273056) [](ancestors = 111273056)